package.json

```diff
@@ -9,17 +9,17 @@
     "*.js"
   ],
   "devDependencies": {
-    "@types/jest": "^29.4.0",
-    "@types/node": "^18.14.6",
+    "@types/jest": "^29.5.3",
+    "@types/node": "^20.5.2",
     "@typescript-eslint/eslint-plugin": "^5.54.1",
-    "eslint": "^8.35.0",
+    "eslint": "^8.47.0",
     "eslint-config-braintree": "^6.0.0-typescript-prep-rc.2",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^29.6.2",
-    "jest-environment-jsdom": "^29.6.2",
-    "prettier": "^2.8.4",
+    "eslint-plugin-prettier": "^5.0.0",
+    "jest": "^29.6.3",
```

Comment:
The only jest library I've been updating has been the @types/jest since that will still be used with Vitest. The others I've been ignoring since they'll get removed soon.